src/main/java/com/knu/buga1chuk/serialization/json/ReadPersonJsonDemo.java

```diff
@@ -4,12 +4,18 @@
 import com.knu.buga1chuk.model.Person;
 import com.knu.buga1chuk.serialization.service.PersonJsonService;
 import com.knu.buga1chuk.service.PersonService;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
 
 import java.io.File;
 import java.util.List;
 
 public class ReadPersonJsonDemo {
 
+    private static final Logger LOGGER = LoggerFactory.getLogger(ReadPersonJsonDemo.class);
+
+
     public static void main(String[] args) {
```

Comment:
Remove `System.out.println` from the codebase

Reply:
Done